CHANGELOG.md

```diff
@@ -1,5 +1,100 @@
 [//]: # (current developments)
 
+## 4.12.0 (2022-03-XX)
+
+### Enhancements
+
+* Add support for libmamba integrations. (#11193)
+
+  This is a new **experimental and opt-in** feature that allows use of the
+  new [conda-libmamba-solver](https://github.com/conda-incubator/conda-libmamba-solver)
+  for an improved user experience, based on the libmamba community project -
+  the library version of the [mamba package manager](https://github.com/mamba-org/mamba).
+
+  This is an  which means you'll need to follow these steps:
```

Comment:
```suggestion
  Since the libmamba solver is provided as a standalone package you'll need to
  follow these **extra** steps to install & use the experimental solver:
```

Reply:
Wow, not sure what happened there with my English 🤣